src/web/app/components/response-time-series-chart/response-time-series-chart.component.ts

```diff
@@ -19,30 +19,31 @@ export class ResponseTimeSeriesChartComponent implements OnInit {
 
   @Input()
   model: ResponseTimeSeriesChartModel = {
-    durationMinutes: 60,
-    intervalMinutes: 1,
+    durationMinutes: 1440,
+    intervalSeconds: 10,
```

Comment:
changed default duration to 10s (again for testing), 1 minute is too long to observe anything when testing

Reply:
Can ask if this interval is the FE ping BE interval, not the interval param to BE?